### PR TITLE
marvin: Notify skipped tests

### PR DESCRIPTION
**Pull request by davidjumani**
Smoke tests completed. 94 look OK, 1 have errors, 1 did not run
Only failed and skipped tests results shown below:

Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_01_volume_usage | `Failure` | 194.15 | test_usage.py
all_test_create_network | `Skipped` | --- | test_create_network.py

### CODE REVIEW

File: Ansible/roles/marvin/templates/smoketests.sh.j2 [hunk 1]
```diff
@@ -79,24 +81,31 @@ for file in $FILES; do
...
   else
-    PASSES=$((PASSES+1))
-    echo "test $TESTFILENAME in {{ env_name_clean }} looks OK"
-    {% if use_hipchat %}
-      hipchat --action sendNotification --room "Marvin Notifications" --messageFormat "html" --colour "gray" --message "test $TESTFILENAME in {{ env_name_clean }} looks <span style="color:#2F962F">OK</span>. $PASSES of $counter good so far.<br>Tests have taken $all_tests_elapsed hours so far. $tests_left tests to go."
-    {% endif %}
-    {% if use_slack %}
-      slackmsg "Test $TESTFILENAME looks *OK*.\n$PASSES of $counter good so far.\nTests have taken $all_tests_elapsed hours so far. $tests_left tests to go." "good"
-    {% endif %}
+    echo "test $TESTFILENAME in ref-trl-3629-k-Mr8-daan-hoogland did not run"
```
Comment (DaanHoogland) [inline Ansible/roles/marvin/templates/smoketests.sh.j2:106]:
```suggestion
    echo "test $TESTFILENAME in {{ env_name_clean }} did not run"
```

**Review by DaanHoogland — COMMENTED** [review]
almost proud

**Review by rohityadavcloud — APPROVED** [review]
Needs testing LGTM

File: Ansible/roles/marvin/files/xunit-reader.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def parse_reports(file_path_list):
 
     for test in tests:
         print("%s | %s | %.2f | %s" % (test[0], test[1], float(test[2]), test[3]))
-    print("")
+    # print("")
```
Comment (DaanHoogland) [inline Ansible/roles/marvin/files/xunit-reader.py:106]: let's delete

**Review by DaanHoogland — APPROVED** [review]

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**